paasta_tools/kubernetes_tools.py

```diff
@@ -2144,6 +2150,12 @@ def get_pod_template_spec(
         docker_volumes = self.get_volumes(
             system_volumes=system_paasta_config.get_volumes()
         )
+        if not self.has_bulkdata():
```

Comment:
rather than excluding the volume from the default volumes, can you instead remove `/nail/bulkdata` from the default volumes, and then add when `has_bulkdata` is `True`? It just removes some of the surprise from what `default_volumes` really does

Reply:
i think we'd need to roll this out first before removing `/nail/bulkdata` from the default mounts - otherwise we'll potentially be removing the bulkdata mount from services that need it but are on a cluster where this paasta version has not yet rolled out

Reply:
I've changed this and added details in the tech spec for how we should roll this out